_data/hosts.yml

```diff
@@ -802,6 +802,38 @@
             version: 5.5.1
             semver: 5.5.1
 -
+    name: KingHost
+    url: 'https://kinghost.com.br/'
+    type: shared
+    default: 53
+    versions:
+        53:
+            phpinfo: null
+            patch: 29
+            version: 5.3.29
+            semver: 5.3.29
+        54:
+            phpinfo: null
+            patch: 33
+            version: 5.4.33
+            semver: 5.4.33
+        55:
+            phpinfo: null
+            patch: 32
+            version: 5.5.32
+            semver: 5.5.32
+        56:
+            phpinfo: null
+            patch: 18
+            version: 5.6.18
+            semver: 5.6.18
+        70:
+            phpinfo: null
+            patch: 3
+            version: 7.0.3
+            semver: 7.0.3
+-
```

Comment:
Hi buddy, whats up? :) I tested this PHP versions on Kinghost today. All worlked well.

Thanks.

Cheers.